.github/workflows/release-build.yml

```diff
@@ -334,7 +334,7 @@ jobs:
           rustup target add wasm32-unknown-unknown
 
       - name: Install wasm-pack
-        run: curl https://rustwasm.github.io/wasm-pack/installer/init.sh -sSf | bash -s -- -y
+        run: CARGO_HOME=/root/.cargo cargo install wasm-pack --version 0.10.3
```

Comment:
why did you pick this version? I see problems with compilation on windows with this specific version.
https://github.com/rustwasm/wasm-pack/issues/1157

Reply:
It's the latest version that is compatible with our toolchain. We shouldn't need to worry about Windows errors as we don't build for Windows.